docs/components/ComponentPropsTable.tsx

```diff
@@ -43,6 +43,27 @@ export const ComponentPropsTable = ({ data }: ComponentPropsTableProps) => (
 		</TableHead>
 		<TableBody>
 			{Object.values(data.props).map((prop) => {
+				let defaultValue = prop.defaultValue?.value;
+
+				if (defaultValue) {
+					if (
+						prop.type.name === 'boolean' ||
```

Comment:
This fixes `"false"` and `"[]"` being rendered in examples like this https://design-system.agriculture.gov.au/pr-preview/pr-1582/components/file-upload/code